src/tools/run-make-support/src/diff/mod.rs

```diff
@@ -97,6 +100,15 @@ impl Diff {
             .to_string();
 
         if !output.is_empty() {
+            // If we can bless (meaning we have a file to write into and the `RUSTC_BLESS_TEST`
+            // environment variable set), then we write into the file and return.
+            if let Some(ref expected_file) = self.expected_file {
+                if std::env::var("RUSTC_BLESS_TEST").is_ok() {
+                    println!("Blessing `{}`", expected_file.display());
+                    std::fs::write(expected_file, actual).unwrap();
```

Comment:
Suggestion: we can use `fs_wrapper::write()` for this now that `fs_wrapper` helpers are merged into the support lib.

Reply:
I also did it for other fs calls in the same file.